bazel/external/quiche.BUILD

```diff
@@ -3408,3 +3399,37 @@ envoy_cc_test(
         ":quic_platform_test_mem_slice_vector_lib",
     ],
 )
+
+envoy_cc_library(
```

Comment:
nits: mind placing the new build target in alphabetical order?

Reply:
Of course, as much as it's possible (it's not exactly sorted currently, but close).